assets/scripts/build_gallery.py

```diff
@@ -306,11 +311,14 @@ def get_expectation_instances(expectations_info):
                 continue
 
         try:
-            expectation_instances[
-                expectation_name
-            ] = great_expectations.expectations.registry.get_expectation_impl(
-                expectation_name
-            )()
+            expectation_class = (
+                great_expectations.expectations.registry.get_expectation_impl(
+                    expectation_name
+                )
+            )
+            # instantiate expectation
```

Comment:
nit: I don't think this comment is adding any value. If anything I'd just rename `s/exp/expectation` to over communicate it's an instance of an expectation.